.pyspelling.yml

```diff
@@ -0,0 +1,49 @@
+matrix:
+- name: Python Source
+  aspell:
+    lang: en
+    d: en_GB
+  dictionary:
+    wordlists:
+    - wordlist-custom.txt
```

Comment:
please suggest a better place/name for putting the wordlist :)

Reply:
I'm fine with is being in root.